xchange-stream-btcmarkets/src/main/java/info/bitrich/xchangestream/btcmarkets/BTCMarketsStreamingMarketDataService.java

```diff
@@ -29,6 +31,12 @@ private OrderBook handleOrderbookMessage(BTCMarketsWebSocketOrderbookMessage mes
     return BTCMarketsStreamingAdapters.adaptOrderbookMessageToOrderbook(message);
   }
 
+
+  public Ticker handleTickerMessage(BTCMarketsWebSocketTickerMessage message) {
```

Comment:
is this extra method necessary?

Reply:
I think we do need it.

Reply:
why? is using the already single line body not sufficent?

Reply:
Still Pending - Will review it later, Busy with another assignment.